ui/components/multichain/nft-item/nft-item.js

```diff
@@ -25,9 +28,10 @@ export const NftItem = ({
   networkSrc,
   tokenId,
   onClick,
-  clickable = false,
+  clickable,
```

Comment:
Why did we change this default value here?

Reply:
Because we want to make it clickable by default as it's gonna have a show button now in NFT Default Image